lib/hanami/cli/commands/app/generate/view.rb

```diff
@@ -18,10 +18,10 @@ class View < App::Command
             DEFAULT_FORMAT = "html"
             private_constant :DEFAULT_FORMAT
 
-            # TODO: make engine configurable
-
             argument :name, required: true, desc: "View name"
             option :slice, required: false, desc: "Slice name"
+            option :template, required: false, desc: "Template engine to use (officially supported: erb, haml, slim)",
```

Comment:
What should happen if an engine outside the list is called? Currently you can do `--template md` and it will create a file with `.md`  extension and with default HTML (from ERb engine). Which is not entirely wrong, but maybe the list should be restricted and produce error if user specifies unsupported engine?

This also brought the question if this should not be extensible in some way, so gems could hook into it and provide own base templates for their engines (for example `hanami-phlex`).